stats/cloud/logs.go

```diff
@@ -0,0 +1,193 @@
+/*
+ *
+ * k6 - a next-generation load testing tool
+ * Copyright (C) 2020 Load Impact
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package cloud
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/url"
+	"sort"
+	"strconv"
+	"time"
+
+	"github.com/gorilla/websocket"
+	"github.com/mailru/easyjson"
+	"github.com/sirupsen/logrus"
+)
+
+//easyjson:json
+type msg struct {
+	Streams        []msgStreams        `json:"streams"`
+	DroppedEntries []msgDroppedEntries `json:"dropped_entries"`
+}
+
+//easyjson:json
+type msgStreams struct {
+	Stream map[string]string `json:"stream"`
+	Values [][2]string       `json:"values"` // this can be optimized
+}
+
+//easyjson:json
+type msgDroppedEntries struct {
+	Labels    map[string]string `json:"labels"`
+	Timestamp string            `json:"timestamp"`
+}
+
+func getLevelsStr(level string) ([]string, error) {
```

Comment:
As far as I can see, you no longer need that function, it's only used in its own test.

Reply:
this will drop the test coverage ... further :(